lib/bolt/config.rb

```diff
@@ -36,6 +36,7 @@ class Config
 
     OPTIONS = {
       "apply_settings"           => "A map of Puppet settings to use when applying Puppet code",
+      "cleanup"                  => "Whether to clean up temporary files created on targets",
```

Comment:
The changes in `Bolt::Config` look like they're not needed - I'm guessing this is from before deciding it'd be a transport-level option?

Reply:
Whoops yeah. I'll remove that.